api/models/site.js

```diff
@@ -52,11 +64,10 @@ function toJSON() {
   object.createdAt = object.createdAt.toISOString();
   object.updatedAt = object.updatedAt.toISOString();
 
-  const s3Config = config.s3;
-  object.siteRoot = `http://${s3Config.bucket}.s3-website-${s3Config.region}.amazonaws.com`;
```

Comment:
removed the S3 `siteRoot` from the serialized `site` model since we never want to link to the S3 bucket URL